### PR TITLE
updated airbus jenksinfile

### PR DESCRIPTION
**Pull request by mohansgithub**

### CODE REVIEW

**Review by Hemachandra-423 — APPROVED** [review]
This looks good to merger

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,42 @@
+pipeline{
```
Comment (vijindas) [inline Jenkinsfile:3]: add author name

**Review by vijindas — CHANGES_REQUESTED** [review]
fix the comments

File: Jenkinsfile (hunk 1 above)
Comment (mohansgithub) [reply Jenkinsfile:3]: Fixed with latest commit.

**Review by mohansgithub — COMMENTED** [review]
please check

**Review by vijindas — APPROVED** [review]

**Review by vijindas — APPROVED** [review]
Approved

**Review by Hemachandra-423 — APPROVED** [review]

**Comment by mohansgithub** [comment]
Merge

**Comment by mohansgithub** [comment]
Thanks

**Comment by mohansgithub** [comment]
Done

**Pull request merged**